src/partner-access/partner-access.controller.ts

```diff
@@ -17,7 +17,10 @@ import { PartnerAccessService } from './partner-access.service';
 export class PartnerAccessController {
   constructor(private readonly partnerAccessService: PartnerAccessService) {}
 
-  @ApiBearerAuth()
+  @ApiBearerAuth('access-token')
+  @ApiOperation({
+    description: 'Creates an unassigned but the unique code that will be shared with the user',
```

Comment:
Can we tweak this wording to be more like 
`Creates an unassigned partner access record, with a unique code that will be shared with the user to register with. The payload sets the features enabled for this partner access / future user.`

It's good to be clear about these being `PartnerAccess` records, rather than just codes